awx/ui/client/src/instance-groups/list/instance-groups-list.controller.js

```diff
@@ -10,6 +33,7 @@ export default ['$scope', '$filter', '$state', 'Alert', 'resolvedModels', 'Datas
         init();
 
         function init(){
+            $rootScope.breadcrumb.instance_group_name = instanceGroup.get('name');
```

Comment:
@marshmalien 

I tried a few test strings for xss and it sure _seems_ like nothing gets through.

That said, I didn't find anything in the [breadcrumb code](https://github.com/ansible/awx/tree/devel/awx/ui/client/src/bread-crumb) that _explicitly_ sanitizes all content. 

If you can confirm that the IG name will be sanitized before it is displayed in the breadcrumb (perhaps in a different part of the code that I didn't check?) then I think this is fine as is. Otherwise, please go ahead and `$filter('sanitize')(instanceGroup.get('name'))`